gulpfile.js

```diff
@@ -19,8 +20,9 @@ gulp.task('clean', function() {
   ]);
 });
 
-gulp.task('lint', function() {
-  return gulp.src(['src/*.gs', 'test/**/*.js'])
-    .pipe(jshint())
-    .pipe(jshint.reporter(stylish));
+gulp.task('lint', () => {
+  return gulp.src(['src/*.gs', 'samples/*.gs', 'test/**/*.js', '!node_modules/**'])
+    .pipe(eslint())
```

Comment:
Note: Please be consistent next time with **4** space statement continuation indentation.

Reply:
Done